index.css

```diff
@@ -106,6 +106,11 @@
     font-size: 2rem;
     margin: 0;
     line-height: 138%;
+    cursor: pointer;
```

Comment:
```suggestion
    cursor: pointer;
    display: inline-block;
```

Reply:
Hey, I had a look at the code and there's a bit of a weirdness about when you hover over the element it triggers the hover state, even if the text isn't taking up all the space.

![runpkg1](https://user-images.githubusercontent.com/21056165/58713161-fee32c00-83b9-11e9-876b-b5c4ecb58e9a.gif)

How do you feel about making the h1 inline-block so that only when you're hovering over the text it triggers the hover state? 😄 

![runpkg2](https://user-images.githubusercontent.com/21056165/58713379-56819780-83ba-11e9-951f-78e4560a3069.gif)

Reply:
Thanks @kiraarghy, looks good to me 👍

Reply:
@t1mwillis could you commit the suggestion to your PR, I don't have permission?